app/src/controllers/log/selectors.js

```diff
@@ -77,37 +77,12 @@ export const querySelector = createQueryParametersSelector({
   defaultSorting: DEFAULT_SORTING,
 });
 
-export const historyItemsSelector = createSelector(
-  historyEntriesSelector,
-  logItemIdSelector,
-  (entriesFromState, logItemId) => {
-    if (!entriesFromState.length) return [];
-    const entries = [...entriesFromState].reverse();
-
-    const currentLaunch = entries.pop();
-    if (!currentLaunch) {
-      return [];
-    }
-    const currentLaunchItem = currentLaunch.resources.find((item) => item.id === logItemId);
-    if (!currentLaunchItem) {
-      return [];
-    }
-    const historyItems = entries.map((historyItem) => {
-      const filteredSameHistoryItems = historyItem.resources.filter(
-        (item) => item.uniqueId === currentLaunchItem.uniqueId,
-      );
-
-      return {
-        ...normalizeHistoryItem(historyItem, filteredSameHistoryItems),
-      };
-    });
-
-    currentLaunchItem.launchNumber = currentLaunch.launchNumber;
-    historyItems.push(currentLaunchItem);
+export const historyItemsSelector = createSelector(historyEntriesSelector, (entriesFromState) => {
+  if (!entriesFromState.length) return [];
+  const historyItems = [...entriesFromState[0].resources].map(normalizeHistoryItem);
```

Comment:
No need to create new array in this case:
` const historyItems = entriesFromState[0].resources.map(normalizeHistoryItem);`

Reply:
Yes, you're right! Fixed.